cfgov/jinja2/v1/owning-a-home/closing-disclosure/index.html

```diff
@@ -8,41 +14,21 @@
     Closing Disclosure Explainer | Consumer Financial Protection Bureau
 {% endblock %}
 
-{% block metadescription %}Use the CFPB's tool to double-check that all the details about your loan are correct on your Closing Disclosure.{% endblock metadescription %}
+{% block metadescription %}
```

Comment:
Interesting. This calls to my attention that `metadescription` is not a block that is defined anywhere in the template chain! The block name defined in `base.html` is `desc`. This change needs to be applied to all three templates being updated in this PR (and also `cfgov/jinja2/v1/rural-or-underserved/index.html`, if you want to include that here for good measure).

```suggestion
{% block desc %}
```

Reply:
Fixed in https://github.com/cfpb/consumerfinance.gov/pull/6446/commits/b132bc522fbfc9b0d9cca0b8c3656bb2c0543656